subsys/bluetooth/host/adv.c

```diff
@@ -269,6 +269,17 @@ void bt_le_ext_adv_foreach(void (*func)(struct bt_le_ext_adv *adv, void *data),
 #endif /* defined(CONFIG_BT_EXT_ADV) */
 }
 
+void bt_adv_reset_adv_pool(void)
+{
+#if defined(CONFIG_BT_EXT_ADV)
+	for (size_t i = 0; i < ARRAY_SIZE(adv_pool); i++) {
+		memset(&adv_pool[i], 0, sizeof(adv_pool[i]));
+	}
+#else
+	memset(&bt_dev.adv, 0, sizeof(bt_dev.adv));
```

Comment:
```suggestion
	(void)memset(&bt_dev.adv, 0, sizeof(bt_dev.adv));
```

For the sake of convention